packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx

```diff
@@ -18,6 +20,8 @@ import { ListItemText } from '../../components/ListItemText';
 import { useNavigateBack } from '../../hooks';
 import { useWallet } from '../../providers';
 
+const defaultTabWidth = 892;
```

Comment:
Let's reduce it to a more mobile-centric value like the default `sm` breakpoint.

Reply:
Sure, we don't want to consider any tablet users?

Reply:
Nice question, tablets resolution can be high to identify it as a tablet or not correctly. They can still select WalletConnect, there are no restrictions on that.

Reply:
Ahh got it. Makes sense